### PR TITLE
Upgrade all packages

### PR DESCRIPTION
**Pull request by keithamus**
This upgrades known dependencies to their latest versions, and makes a few small changes to the `tsconfig` and `fetch_request` files to upgrade the failing TypeScript build.

### CODE REVIEW

**Comment by keithamus** [comment]
The tests are failing because `intern` comes pre-baked with its own copy of ChromeDriver, the ChromeDriver it uses only works with Chrome 99.

GitHub Actions runners use Chrome 98 right now (https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#browsers-and-drivers). They also come with ChromeDriver which works with 98. It should be possible to set an environment flag to get `intern` to use the built in ChromeDriver... somehow :joy:

**Comment by dhh** [comment]
Could we lock intern to a version that has the prebaked 98 for now?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,6 +20,15 @@ jobs:
...
 
+    - name: Set Chrome Version
+      run: |
+        CHROMEVER="$(chromedriver --version | cut -d' ' -f2)"
+        echo "Actions ChromeDriver is $CHROMEVER"
+        CONTENTS="$(jq '.tunnelOptions.drivers[0].name = "chrome"' < intern.json)"
+        CONTENTS="$(echo ${CONTENTS} | jq --arg chromever "$CHROMEVER" '.tunnelOptions.drivers[0].version = $chromever')"
+        echo "${CONTENTS}" > intern.json
+        cat intern.json
+
```
Comment (keithamus) [inline .github/workflows/ci.yml:31]: This script uses the actions environment to determine the correct Chrome version, and writes into Intern.json the version to use. I think intern still downloads it's own copy of ChromeDriver, but at least it gets the tests _running_ :joy:

Comment (dhh) [reply .github/workflows/ci.yml:31]: Yak riders, saddle up!

**Pull request merged**